dask/dataframe/tests/test_dataframe.py

```diff
@@ -3177,14 +3177,22 @@ def test_apply():
 @pytest.mark.parametrize("convert_dtype", [None, True, False])
 def test_apply_convert_dtype(convert_dtype):
     """Make sure that explicit convert_dtype raises a warning with pandas>=2.1"""
-    df = pd.DataFrame({"x": [1, 2, 3, 4], "y": [10, 20, 30, 40]})
+    df = pd.DataFrame({"x": [2, 3, 4, 5], "y": [10, 20, 30, 40]})
```

Comment:
Is this change meaningful? It looks fine, it's just not clear to me why it's being made

Reply:
Yes! _meta_nonempty is [1, 1]. We can't efficiently check if we are still calling apply with ``_meta_nonempty``, if we have the same element in our DataFrame.

Reply:
Ah, I see. Thanks for clarifying